doc/rho.1

```diff
@@ -42,71 +40,106 @@ The following sections cover these commands in more detail.
 .PP
 .B rho auth add --name=
 .I name
-.B --username
-.I username=
+.B --username=
+.I username
+.B [--password]
 .B [--sshkeyfile=
 .I key_file
 .B ]
+.B [--sudo-password]
+.B [--vault=
+.I vault_file
+.B ]
 .PP
 .TP
 --name=name
 This required argument sets the name of the new authentication credentials entry. This should be descriptive, such as identifying the user or server it relates to. For example, "server1-rhouser". It should never contain the actual password, as this name may be logged or printed during rho execution.
 .PP
 .TP
 --username=username
-This required argument contains the username of the SSH identity will use to bind to the server. This parameter prompts for the password for the user identity.
+This required argument contains the username of the SSH identity will use to bind to the server.
+.PP
+.TP
+--password
+The argument prompts for the password for the --username identity.
 .PP
 .TP
 --sshkeyfile=key_file
-Optionally, this contains the path and file name of the file containing the SSH key issued for the --username identity.
-
+Optionally, this contains the path and filename of the file containing the SSH key issued for the --username identity.
 .PP
-The information given in an auth entry -- such as a password, SSH keys, or even the username -- may change. For example, network security may require passwords to be updated every few months. The auth entry can be edited to change the SSH credential information. The parameters for 'rho auth edit' are the same as those for 'rho auth add'.
+.TP
+--sudo-password
+The argument prompts for the password to be used with those commands that run on remote systems utilizing sudo.
+.PP
+.TP
+--vault=vault_file
+This contains the path and filename of the file containing the vault password. If this option is used the file should be limited in access on the system.
+.PP
+
+The information given in an auth entry -- such as a password, sudo password, SSH keys, or even the username -- may change. For example, network security may require passwords to be updated every few months. The auth entry can be edited to change the SSH credential information. The parameters for 'rho auth edit' are the same as those for 'rho auth add'.
 .PP
 .B rho auth edit --name=
 .I name
 .B --username=
 .I username
 .B [--sshkeyfile=
 .I key_file
-.B ] --password
-
+.B ]
+.B [--password]
+.B [--sudo-password]
+.B [--vault=
+.I vault_file
+.B ]
 .PP
 .B
-Listing and Showing auth Entries
+Listing and Showing Auth Entries
 .PP
-The 'rho auth list' command returns the details for every auth entry configured for rho. This output includes the name, username, password, and auth type for each entry.
+The 'rho auth list' command returns the details for every auth entry configured for rho. This output includes the name, username, password, ssh keyfile and sudo password for each entry. Passwords are masked if provided, if not they will appear as ‘None’.
```

Comment:
s/None/null ? None is familiar for someone that knows Python but since the output has null as the value I think it would be better to state that instead.

Reply:
updated to use `null`. Thanks for the correction, I was going from memory.